### DISCUSSION TITLE
Disable "enter" as search in quickfix list

### DISCUSSION THREAD

**Discussion by esn89** [folke/flash.nvim#383] [Q&A]
I have <CR> mapped as "begin to find stuff in flash.nvim".  

Only problem is when I am in quickfix list I have to select an item using <CR>.  How can I make it so that binding ignores quickfix?

**Comment by vandalt** [comment]
I'll repost my comment from #435 in case someone finds this post but not the issue:

> The following should work:
> 
> ```lua
> vim.api.nvim_create_autocmd("FileType", {
>   pattern = "qf",
>   callback = function() vim.keymap.set('n', '<CR>', '<CR>', { buffer = true }) end,
> })
> ```
> 
> Inspired from [mini.jump2d](https://github.com/nvim-mini/mini.nvim/blob/6a224ddd28b4b21a488d6d87e1c2643a029caaa4/lua/mini/jump2d.lua#L784).
